bridge/engine/AudioForwarderReceiveJob.cpp

```diff
@@ -132,10 +132,21 @@ void AudioForwarderReceiveJob::run()
     const auto rtpHeaderExtensions = rtpHeader->getExtensionHeader();
     if (rtpHeaderExtensions)
     {
+        auto c9infoExtId = _ssrcContext._rtpMap._c9infoExtId.isSet() ? _ssrcContext._rtpMap._c9infoExtId.get() : 0;
+
         int32_t audioLevel = -1;
+        utils::Optional<bool> is_ptt;
 
+        // Use one loop to check both extensions for speed up.
```

Comment:
Remove comments that are not extremely necessary to explain what is happening. Code should speak for itself.

Reply:
Removed.